internal/config/config.go

```diff
@@ -75,7 +75,7 @@ var DefaultConfig Config = Config{
 	AWSRegion:               "us-east-1",
 	AWSSecretAccessKey:      os.Getenv("CW_AWS_SECRET_ACCESS_KEY"),
 	CloudConnectorClientID:  "config-manager",
-	CloudConnectorHost:      flagvar.URL{Value: url.MustParse("http://cloud-connector:8080")},
+	CloudConnectorHost:      flagvar.URL{Value: url.MustParse("http://localhost:8080")},
```

Comment:
Any specific reason for all these changes in config.go? We are defining this in Makefile - [here](https://github.com/RedHatInsights/config-manager/blob/master/Makefile#L43).

Reply:
Why are these added as default instead of localhost?  Where are these hosts defined?

Reply:
I am not sure, that is the reason I did not changed it initially and instead added flags in the Makefile command.

Reply:
Yes, that's why I changed it to `localhost` because accessing API using curl would fail if we use these hosts.